mantidproject/mantid
# Update muon interface docs

To be merged after #25758

**Description of work.** 
* Written up documentation for the Muon phase tab
* Written up documentation for the Muon analysis fitting tab
* Written up documentation for the Muon analysis 2 interface
* Updated the transform tab on frequency domain analysis interface to reflect the moving of the phase tab.
* Replaced of sceenshots for the muon and frequency domain analysis and documentation to reflect new layout

**Report to:** AnthonyLim23

**To test:**

1. Build mantid documentaion.
2. In the documentation look in interfaces.
3. check that the following html files are legible and suitable for documentation: 
* Muon Analysis 2 
* Frequency Domain Analysis 
* muon_fitting_tab
* muon_grouping_tab
* muon_home_tab
* muon_phase_table_tab
* muon_transform_tab

Fixes #25766  #25669 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
